databases/week2/hyf_lesson2_homework.sql

```diff
@@ -0,0 +1,93 @@
+-- Part 1: Working with tasks
+INSERT INTO
+  task(
+    title,
+    description,
+    created,
+    updated,
+    due_date,
+    status_id
+  )
+VALUES
+  (
+    "studying",
+    "study with coding database",
+    "2022-02-07",
+    "2022-02-07",
+    "2022-06-01",
+    1
+  );
+UPDATE
+  task
+SET
+  title = "prepare for week3 database"
+WHERE
+  id = 36;
+UPDATE
+  task
+SET
+  due_date = "2022-01-01"
+WHERE
+  id = 36;
+UPDATE
+  task
+SET
+  status_id = 2
+WHERE
+  id = 36;
+UPDATE
+  task
+SET
+  status_id = 3
+WHERE
+  id = 1;
+DELETE FROM
+  task
+WHERE
+  id = 36;
+--   Part 3: More queries
+SELECT
+  task.title,
+  user.email
+FROM
+  task
+  JOIN user_task ON task.id = user_task.task_id
+  JOIN user ON user.id = user_task.user_id
+WHERE
+  user.email LIKE "%@spotify.com";
+SELECT
```

Comment:
its nice to assign the alias, just let you know its not necessary, I know if you dont the VSCode extension will show wrong result, but thats just the VSCode extension, if you use other IDE, it shows correct result